src/Pagerfanta/Adapter/AdaptersAdapter.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Pagerfanta package.
+ *
+ * (c) Pablo Díez <pablodip@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Pagerfanta\Adapter;
+
+class AdaptersAdapter implements AdapterInterface
```

Comment:
The name of the class is a bad name, as it does not describe how the adapters are combined. I can think of at least 2 ways to implement an adapter bring other adapters together:

- considering that it paginates an AppendIterator of the different iterators (what you implement it)
- considering that it paginates a MultipleIterator looping over the different iterators in parallel